### PR TITLE
SUMO-192150: FGP release doc

### PR DESCRIPTION
**Pull request by ayiusumo**
SUMO-192150: FGP release doc

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ## 2.16.0 (Unreleased)
 
+FEATURES:
```
Comment (sumovishal) [inline CHANGELOG.md:3]: Add it to `2.15.1 (Unreleased)` release.

File: website/docs/r/monitor_folder.html.markdown [hunk 2]
```diff
@@ -8,6 +8,8 @@ description: |-
 # sumologic_monitor_folder
 
 Provides the ability to create, read, delete, and update folders for [Monitors][1].
+If Fine Grain Permission (FGP) feature is enabled with Monitors Content at one's Sumo Logic account,
```
Comment (sumovishal) [inline website/docs/r/monitor_folder.html.markdown:11]: Prefix it with "->" to create a note. 
```
-> If Fine...
```

File: CHANGELOG.md (hunk 1 above)
Comment (ayiusumo) [reply CHANGELOG.md]: updated the file as suggested.

File: website/docs/r/monitor_folder.html.markdown (hunk 2 above)
Comment (ayiusumo) [reply website/docs/r/monitor_folder.html.markdown]: updated the file as suggested.

**Review by sumovishal — APPROVED** [review]

**Pull request merged**